app/js/spatial/geo.js

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Daniel W. Steinbrook.
 // with many thanks to ChatGPT
 
+const turf = require('@turf/turf');
+
+module.exports = { createBoundingBox , enumerateTilesInBoundingBox, getLocation,  watchLocation, geoToXY};
```

Comment:
This syntax might work in Node from the command line, but breaks in a browser. I'd think there should be a way to get Mocha to work while keeping ES6 module syntax like we have now. See: https://stackoverflow.com/questions/46487307/is-it-possible-to-use-es6-modules-in-mocha-tests